docs/learn/learn-common-good-chains.md

```diff
@@ -59,19 +59,17 @@ system parameters or triggering an upgrade.
 Because public utility chains add functionality beyond the scope of the Relay Chain, they will
 likely be approved by the network stakeholders only in rare scenarios.
 
-## Common Good Chains in Development
+## Common Good Chains
 
 ### Statemint
 
-[Statemint](https://github.com/paritytech/cumulus/tree/master/polkadot-parachains/statemint) (and its cousin *Statemine* on Kusama) will
-likely be one of the first common good parachains.
-
-> Statemine is the first common good parachain.
+[Statemint](https://github.com/paritytech/cumulus/tree/master/polkadot-parachains/statemint) 
+(and its cousin *Statemine* on Kusama) are the first common good parachains.
 
 Statemint is a public utility chain in that it adds functionality not available in the Relay Chain, 
-namely the creation and management of assets. Statemint will support both fungible and non-fungible 
+namely, the creation and management of assets. Statemint will support both fungible and non-fungible 
 assets. The chain offers an interface similar to ERC-20 for fungible assets and ERC-721 for non-fungible 
-tokens. These interfaces are in the logic of the chain itself; by encoding this logic directly into the 
+assets. These interfaces are in the logic of the chain itself; by encoding this logic directly into the 
```

Comment:
I see that you wanted to keep it uniform and replace tokens with assets. I just ran a quick online search and I see people referring to ERC-20 for fungible tokens and ERC-721 for Non-fungible tokens. 

In the context of Statemint, which is often referred to as an Assets parachain, I guess it is okay to use tokens and assets interchangeably. No action required.